README.md

```diff
@@ -163,7 +163,7 @@ symlink the `Makefile` to the correct device specific makefile, that means that
 you simply start the build by running:
 
 ```bash
-$ make all run
+$ make all
```

Comment:
`all` should be the default target so `make` should be enough.

Related to this: I think we need another patch to `s/all-clean/clean/` because `all-clean` is just stupid. Everyone expects to find a `clean` target.